igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/service/JenkinsService.java

```diff
@@ -240,8 +243,9 @@ public Build getLatestBuild(String jobName) {
   public QueuedJob queuedBuild(String master, int item) {
     try {
       return circuitBreaker.executeSupplier(() -> jenkinsClient.getQueuedItem(item));
-    } catch (RetrofitError e) {
-      if (e.getResponse() != null && e.getResponse().getStatus() == NOT_FOUND.value()) {
+    } catch (SpinnakerServerException e) {
```

Comment:
Since this re-throws everything that's not a 404, 
```suggestion
    } catch (SpinnakerHttpException e) {
```
and remove the instanceof check